test/suites/positive/positive-suite.test.ts

```diff
@@ -0,0 +1,62 @@
+const operationResult = new Map();
+
+import loadJsonFile from 'load-json-file';
+import OASSchema from '../../../src/oas-parsing/schema/oas-schema';
+import { SuiteConfig } from '../../../src/suites';
+import PositiveSuite from '../../../src/suites/positive/positive-suite';
+
+jest.mock('../../../src/suites/positive/conductors', () => {
+  return {
+    __esModule: true,
+    ValidationConductor: jest.fn().mockImplementation(() => {
+      return {
+        ValidationConductor: jest.fn(),
+        validate: jest.fn().mockResolvedValue(operationResult),
+      };
+    }),
+  };
+});
+
+let suiteConfig: SuiteConfig;
+let oasSchema: OASSchema;
+operationResult.set(
+  '/findTheRing:GET',
+  new Map().set('no-rings-found', {
+    failures: { test: 'mockError' },
+    warnings: {},
+  }),
+);
```

Comment:
It looks like this operationResult is similar or the same as the one used in `spectral-suite.test.ts`. Can this be defined as a fixture and reused in both these tests?

Reply:
Sure

Reply:
From looking over the code I thought this was larger code-wise than what was shown.  It doesn't make sense to move this to create a fixture for 7 lines.